Bottom: e0222666bc16346a134d98b1c36d58d0e6fdc515
Top:    f79471d4897c881fe44cdc09cd18cfb45193ca36
Author: Sergey Alyoshin <alyoshin.s@gmail.com>
Date:   2010-06-02 21:39:55 +0100

Localize need coord message before output


---

diff --git a/src/hid/common/actions.c b/src/hid/common/actions.c
index 9fa129d..d1e87f5 100644
--- a/src/hid/common/actions.c
+++ b/src/hid/common/actions.c
@@ -231,7 +231,7 @@ hid_actionv (const char *name, int argc, char **argv)
     }
 
   if (a->need_coord_msg)
-    gui->get_coords (a->need_coord_msg, &x, &y);
+    gui->get_coords (_(a->need_coord_msg), &x, &y);
 
   if (Settings.verbose)
     {
